hosting_docs/source/backups_dr/2-backups-guide.rst

```diff
@@ -284,10 +284,28 @@ BlobDB backups create a compressed version of the blobdb data directory.
 
 Restoring BlobDB Backups
 ^^^^^^^^^^^^^^^^^^^^^^^^
-
-You can follow the same instructions as for `restoring couchdb <#restoring-couchdb-backups>`_ (extract the backup file into the blobdb data directory: ``/opt/data/blobdb/``\ ). 
-
-The files in the resulting directory should all be owned by the user ``cchq`` (i.e. you should be the ``cchq`` user when extracting the files)
+The BlobDB restore process depends on what BlobDB system you're using.
+- If you're using the default file system BlobDB, the restore process is the same as the couchdb restore process in that it involves extracting the backed up data 
+to the data directory. 
+- If you're using some other (distributed) system you should follow that service's provided instructions on restoration.
+
+The file system BlobDB restore process will be explained below.
+
+# Become the `cchq` user
+  ``` bash
+  $ sudo -iu cchq
+  ```
+
+# Now we need to extract the backup data. The BlobDB backups live in the `/opt/data/backups/blobdb` directory by default (if you have specified a different path
+in the `public.yml` file, it will be there instead).
+  ``` bash
+  $ tar -xf /opt/data/backups/blobdb/blobdb_<version>.gz -C /opt/data/backups/blobdb
+  ```
+
+# Move the data to the `/opt/data/blobdb/` directory.
+  ``` bash
+  $ rsync -avz --delete /opt/data/backups/blobdb/blobdb_<version> /opt/data/blobdb/
+  ```
```

Comment:
I think this is Markdown in a reStructuredText file.

```suggestion

1. Become the ``cchq`` user

   .. code:: bash

       $ sudo -iu cchq

2. Now we need to extract the backup data. The BlobDB backups live in
   the ``/opt/data/backups/blobdb`` directory by default (if you have
   specified a different path in the ``public.yml`` file, it will be
   there instead).

   .. code:: bash

       $ tar -xf /opt/data/backups/blobdb/blobdb_<version>.gz -C /opt/data/backups/blobdb

3. Move the data to the ``/opt/data/blobdb/`` directory.

   .. code:: bash

       $ rsync -avz --delete /opt/data/backups/blobdb/blobdb_<version> /opt/data/blobdb/

```